src/scss/constants.scss

```diff
@@ -139,6 +139,7 @@ $media-breakpoint--sm: 576px;
 $media-breakpoint--md: 768px;
 $media-breakpoint--lg: 992px;
 $media-breakpoint--xl: 1200px;
+$media-breakpoint--xxl: 1350px;
```

Comment:
Could be useful to add line comments after the `;` on each one of these that adds context. 

Something from these browser lists should do it.
![sparkle-display-sizes-02](https://user-images.githubusercontent.com/79229621/143473651-4a070cb3-65b2-49b5-acec-ea2d32cd7a4b.png)
![sparkle-display-sizes-01](https://user-images.githubusercontent.com/79229621/143473655-bc18282c-353a-4276-a972-caaf57ff5f40.png)

Reply:
wasn't sure if something from that list would work, so I added general namings for those constants, like `tablet`, `mobile`, `tablet-small` etc. f9043d4

Reply:
Naming is OK, was just curious if there was an issue of adding the `//` comments in the same line after `;` as to not disturb the sequence